.github/workflows/doc-build.yml

```diff
@@ -0,0 +1,61 @@
+name: Quarkus Documentation CI
+
+on:
+  push:
+    paths:
+      - 'docs/src/main/asciidoc/**'
+  pull_request:
+    paths:
+      - 'docs/src/main/asciidoc/**'
+
+
+jobs:
+  build-doc:
+    name: "Documentation Build"
+    runs-on: ubuntu-latest
+    # Skip draft PRs and those with WIP in the subject, rerun as soon as its removed
+    if: "github.event_name != 'pull_request' || ( \
+             github.event.pull_request.draft == false && \
+             github.event.pull_request.state != 'closed' && \
+             contains(github.event.pull_request.title, 'wip ') == false && \
+             contains(github.event.pull_request.title, '[wip]') == false && \
+             (
+               github.event.action != 'edited' || \
+               contains(github.event.changes.title.from, 'wip ') || \
+               contains(github.event.changes.title.from, '[wip]') \
+             ) \
+           )"
```

Comment:
I think I would run it anyway in this case so that we can have early feedback on the doc. Usually, that's not the doc that is the issue.

Reply:
Yes, this could be a good thing as it could catch Maven mis-configuration earlier. As I launch a build from the root directory it still reads all pom even if it only launch the build on the doc thanks to `-pl docs`.

Reply:
Done